tools/fuzz/fuzz.cpp

```diff
@@ -107,6 +107,8 @@ Options (in lexicographical order):
                provided if the tool is invoked in fuzzing mode; incompatible
                with replay and shrink modes.  The file should be empty if no
                donors are to be used.
+  --enable-all-passes
+               TODO explain.
```

Comment:
TODO

Reply:
Oops!  I have a regex that looks for unaddressed TODOs, but I only ran it over source/fuzz, not tools/fuzz.  Glad you spotted this!!  Fixed.